Fixes Issue : 316 :: https://github.com/mesos/kafka/issues/316

NNSatyaKarthik opened the pull request:
updated gradle build to 4.1
updated de.undercouch.download to version 3.2.0 and made changes in jar task respectively
added dist as precise in .travis.yaml as going forward travis is using trusty as the default run environment.

steveniemitz reviewed (approved):
lgtm, lets see how travis ci likes it.

steveniemitz commented:
Thanks for this one!  I'll look over your other one when I can also

NNSatyaKarthik committed:
updated gradle build to 4.1
updated de.undercouch.download to version 3.2.0 and made changes in jar task respectively
added dist as precise in .travis.yaml as going forward travis is using trusty as the default run environment.

NNSatyaKarthik commented:
Sure!. 👍
